docs/index.rst

```diff
@@ -12,6 +12,17 @@ NI-XNET Python Documentation
    :maxdepth: 3
    :caption: API Reference:
 
+   session
+   base
+   frames
+   signals
+   types
+   errors
+   constants
+   collection
+   intf
+   j1939
```

Comment:
[DAQmx only included top level modules](https://raw.githubusercontent.com/ni/nidaqmx-python/master/docs/index.rst) and left things like the following to the sub pages
- frames
- signals
- intf
- j1939

My hope is we can get away without directly listing
- base
- collection